app/models/piece.rb

```diff
@@ -31,60 +31,32 @@ def move_to!(x, y)
     end
   end
 
-# Returns true/false if a piece is obstructed in its movement.
-# Errors for movements not diagonal, vertical or horizontal
-  def is_obstructed?(x, y)
-  # Immediately return false if moving piece is a Knight, which is exempt from obstruction
+  # Returns true/false if a piece is obstructed in its movement.
+  def obstructed?(x, y)
+    # Immediately return false if moving piece is a Knight, which is exempt from obstruction
```

Comment:
I'm not married to the idea of changing it. We can keep it the same. I only changed it to appease Rubocop. Surprisingly, Rubocop created a citation or violation (or whatever they're called) to remove the "is_" part of the method name. Not really looking into the reason behind this I just went ahead and changed it. I assumed that Rubocop was suggesting this because perhaps the word "obstructed?" may be adequate for most uses. I'll try to do some digging into why Rubocop recommends this just for my own curiosity. But like I said we can keep it as is if it makes for better readability.